client/dashboard/sites/index.tsx

```diff
@@ -43,11 +43,7 @@ const DEFAULT_FIELDS: Field< Site >[] = [
 		id: 'URL',
 		label: __( 'URL' ),
 		enableGlobalSearch: true,
-		render: ( { item }: { item: Site } ) => (
-			<ExternalLink href={ item.URL } style={ { overflowWrap: 'anywhere' } }>
-				{ new URL( item.URL ).hostname }
-			</ExternalLink>
-		),
+		render: ( { item }: { item: Site } ) => new URL( item.URL ).hostname,
```

Comment:
This is a weird change, do you want to remove the urls?

Reply:
Check Figma, they want to remove them

Reply:
It's a bit weird to have render links that are not clickable to be honest.